tls/ownership/ownership.md

```diff
@@ -0,0 +1,1315 @@
+## Missing Owners
```

Comment:
That looks useful, do we want "known owners" too? I suppose it would be useful for TRT and they would probably be fine with looking those up in JSONs?